pyteal/ast/router.py

```diff
@@ -637,21 +620,28 @@ def method(
             opt_in (optional): The allowed calls during :code:`OnComplete.OptIn`.
             close_out (optional): The allowed calls during :code:`OnComplete.CloseOut`.
             clear_state (optional): The allowed calls during :code:`OnComplete.ClearState`.
+                This argument has been deprecated, and will error on compile time if one wants to access it.
             update_application (optional): The allowed calls during :code:`OnComplete.UpdateApplication`.
             delete_application (optional): The allowed calls during :code:`OnComplete.DeleteApplication`.
         """
         # we use `is None` extensively for CallConfig to distinguish 2 following cases
         # - None
         # - CallConfig.Never
         # both cases evaluate to False in if statement.
+
+        if clear_state is not None:
+            raise TealInputError(
+                "Attempt to register ABI method for clear state program. "
+                "Use Router top level argument `clear_state` instead."
```

Comment:
It looks like you added a link to the other error message but not this one?

Reply:
good find, just add to the error message.